src/pages/regulations/List.vue

```diff
@@ -0,0 +1,64 @@
+<script setup lang="ts">
+import Table from "@/base-components/Table/Table.vue"
+import Thead from "@/base-components/Table/Thead.vue";
+import Tbody from "@/base-components/Table/Tbody.vue";
+import Tr from "@/base-components/Table/Tr.vue";
+import Td from "@/base-components/Table/Td.vue";
+import Th from "@/base-components/Table/Th.vue";
+import { useRegulationsStore } from '@/stores/regulations';
+import { computed, onBeforeMount, ref } from 'vue';
+import { useAlertStore } from "@/stores/alert";
+import Lucide from "@/base-components/Lucide/Lucide.vue";
```

Comment:
liste için mobil görünümde şöyle bir problemimiz var.

![image](https://user-images.githubusercontent.com/6912824/227774449-03f1c1f6-c9b7-45ef-828d-4aeaba464a82.png)